modules/tinymce/src/plugins/lists/main/ts/core/Delete.ts

```diff
@@ -37,7 +37,7 @@ const findNextCaretContainer = (editor: Editor, rng: Range, isForward: boolean,
     }
   }
 
-  while ((node = walker[isForward ? 'next' : 'prev2']())) {
+  while ((node = (isForward ? walker.next() : walker.prev2()))) {
```

Comment:
```suggestion
  const walkFn = isForward ? walker.next : walker.prev2;
  while ((node = walkFn())) {
```
This is a little easier to read? It also saves having to do the ternary evaluation on every loop.

Reply:
Forgot we'd need to bind `this`, so it should actually be this sorry:
```suggestion
  const walkFn = isForward ? walker.next.bind(walker) : walker.prev2.bind(walker);
  while ((node = walkFn())) {
```